source/wp-content/themes/wporg-developer-2023/templates/single-command.html

```diff
@@ -0,0 +1,34 @@
+<!-- wp:template-part {"slug":"header","className":"has-display-contents"} /-->
+
+<!-- wp:group {"tagName":"main","layout":{"type":"constrained","wideSize":"1280px","contentSize":"680px"},"style":{"spacing":{"padding":{"left":"var:preset|spacing|edge-space","right":"var:preset|spacing|edge-space"}}}} -->
+<main class="wp-block-group alignfull" style="padding-left:var(--wp--preset--spacing--edge-space);padding-right:var(--wp--preset--spacing--edge-space)">
+
+	<!-- wp:group {"layout":{"type":"constrained","justifyContent":"left"},"align":"wide","style":{"spacing":{"margin":{"top":"var:preset|spacing|edge-space"}}}} -->
+	<div class="wp-block-group alignwide" style="margin-top:var(--wp--preset--spacing--edge-space)">
+
+		<!-- wp:group {"tagName":"article","style":{"spacing":{"blockGap":"var:preset|spacing|40","margin":{"top":"0px"}}}} -->
+		<article class="wp-block-group" style="margin-top:0px">
+
+			<!-- wp:wporg/code-reference-title {"level":1} /-->
```

Comment:
This doesn't match the designs. Should we be using the same block or a new one? Or styled default Gutenberg blocks? We also need to handle the `<command>` portion:

<img width="812" alt="Screen Shot 2023-02-13 at 10 45 35 AM" src="https://user-images.githubusercontent.com/1657336/218352743-75f5a607-817c-43a5-89e1-e6e5bcb6715e.png">

Reply:
What about the GitHub buttons?

Reply:
> This doesn't match the designs. Should we be using the same block or a new one

I added a [Block: Command Title](https://github.com/WordPress/wporg-developer/pull/211/commits/f0ce3a8f0e49adaa7a09c51ebac30710b27cfc65).

> What about the GitHub buttons?

[Block: Command Github](https://github.com/WordPress/wporg-developer/pull/211/commits/5c0d65765220d83b6940907769059089681b30c1)